src/devtools/useAtomsDevtools.ts

```diff
@@ -117,18 +122,21 @@ export function useAtomsDevtools(name: string, scope?: Scope) {
   let extension: typeof window['__REDUX_DEVTOOLS_EXTENSION__']
 
   try {
-    extension = window.__REDUX_DEVTOOLS_EXTENSION__
+    extension =
+      ((options?.enabled ? options.enabled : __DEV__) &&
```

Comment:
This isn't what we want. It should be `typeof enabled === 'boolean' ? enabled : __DEV__`, to deal with `enabled=false`.


Hmm, it would be much cleaner if we do:
```js
  enabled ?? __DEV__
```
(we should do this in zustand and valtio too.)

Reply:
not putting undefined causes type problems!